game_api.txt

```diff
@@ -365,7 +365,9 @@ The farming API allows you to easily register plants and hoes.
 		steps = 8,                             -- How many steps the plant has to grow, until it can be harvested
 		-- ^ Always provide a plant texture for each step, format: modname_plantname_i.png (i = stepnumber)
 		minlight = 13,                         -- Minimum light to grow
-		maxlight = default.LIGHT_MAX           -- Maximum light to grow
+		maxlight = default.LIGHT_MAX,          -- Maximum light to grow
+		only_seed = false,                     -- If true, plant will only drop seeds. plant name is used as name for seed
```

Comment:
Long line, 80-90 columns max with tab size 4.

Reply:
You're right, the line is very long. So are other lines above.
I'll fix this for my new line. But I hope that there will be a commit that fixes this whole mess before this PR is merged.